src/types/QIX.ts

```diff
@@ -132,12 +131,46 @@ export interface ExtendedDimensionInfo extends EngineAPI.INxDimensionInfo {
   qCardinalities: {
     qHypercubeCardinal: number;
   };
+  columnWidth: ColumnWidth;
+}
+
+export interface ExtendedMeasureInfo extends EngineAPI.INxMeasureInfo {
+  columnWidth: ColumnWidth;
 }
 
-export interface HyperCube extends EngineAPI.IHyperCube {
+export interface ExtendedHyperCube extends Omit<EngineAPI.IHyperCube, "qDimensionInfo" | "qMeasureInfo"> {
```

Comment:
chose to call these `ExtendedXyz`. We might want to remove that for shorter names

Reply:
Do you actually need to omit the dim and measure info here? Since they get overridden anyway

Reply:
Don't know I followed the pattern from sn-table

Reply:
quick google and seems like you get a type error without it, but it does not complain when I try to remove it